libsolidity/AST.h

```diff
@@ -280,13 +280,13 @@ class FunctionDefinition: public Declaration
 {
 public:
 	FunctionDefinition(Location const& _location, ASTPointer<ASTString> const& _name,
-					bool _isPublic,
+					bool _isPublic, bool _isConstructor,
```

Comment:
Just to be consistent with the rest of the function, you could put `bool _isConstructor` to its own line to have all arguments on their own line